experiments/E0/model003/input.json

```diff
@@ -14,7 +14,9 @@
         "model": "MODEL003",
         "model_args": {
             "lambda_dir": 10.0,
-            "target_angle": 0.0
+            "target_angle": 0.0,
+            "cpm_force_mode": "reciprocal",
+            "cpm_update_direction": "source-to-target"
```

Comment:
Just making sure: in my definition "source-to-target" is the **unnormalized** vector from source to target pixel (I defined them as `source-to-target` and `source-to-target-norm`. But to avoid confusion we should perhaps rename them as `source-to-target-unnorm` and `source-to-target-norm`?

Reply:
Either works for me.